src/io/react-native/native-bridge.ts

```diff
@@ -14,9 +37,11 @@ export function makeNativeBridge(
   const list = makePendingList()
   return {
     call(name, ...args) {
-      return new Promise((resolve, reject) => {
+      const promise = new Promise((resolve, reject) => {
```

Comment:
If the native code fails to resolve for some reason, then you would be left with a dangling promise. This would be quite obvious, since any `await` calls would get stuck, and we would never get our desired fetch response / file contents / random number. This is the same as any other async code in JavaScript - there is no limit on how long a promise can take to resolve, including forever.